windows/RNSVG/Utils.h

```diff
@@ -164,6 +171,63 @@ struct Utils {
     }
   }
 
+#ifdef USE_FABRIC
+  static std::string JSValueAsBrushUnits(
+      std::optional<int32_t> const &value,
+      std::string defaultValue = "objectBoundingBox") {
+    if (value.has_value()) {
```

Comment:
nit: could this be switched to something like "return (value.has_value() && value.value() == 1) ? "userSpaceOnUse" : defaultValue;" 

or just returning defaultValue instead of writing out "objectBoundingBox" again! May make it easier to read